diff --git a/node_modules/branch-cordova-sdk/plugin.xml b/node_modules/branch-cordova-sdk/plugin.xml
index 1e6767b..6731306 100644
--- a/node_modules/branch-cordova-sdk/plugin.xml
+++ b/node_modules/branch-cordova-sdk/plugin.xml
@@ -86,7 +86,7 @@ SOFTWARE.
               <source url="https://github.com/CocoaPods/Specs.git"/>
           </config>
           <pods>
-              <pod name="Branch" spec="~> 0.31.3" />
+              <pod name="Branch" spec="~> 0.33.1" />
           </pods>
       </podspec>
   </platform>
diff --git a/node_modules/branch-cordova-sdk/src/index.js b/node_modules/branch-cordova-sdk/src/index.js
index ae8ff7b..7ef5aef 100644
--- a/node_modules/branch-cordova-sdk/src/index.js
+++ b/node_modules/branch-cordova-sdk/src/index.js
@@ -1,7 +1,7 @@
 var exec = require("cordova/exec");
 var deviceVendor =
   typeof window.clientInformation != "undefined" &&
-  typeof window.clientInformation.vendor != "undefined"
+    typeof window.clientInformation.vendor != "undefined"
     ? window.clientInformation.vendor
     : "unknownVendor";
 
@@ -89,6 +89,10 @@ Branch.prototype.initSession = function initSession() {
   return execute("initSession");
 };
 
+Branch.prototype.initIosSession = function initIosSession() {
+  return execute("initIosSession", []);
+};
+
 Branch.prototype.setRequestMetadata = function setRequestMetadata(key, val) {
   if (!key || typeof key !== "string") {
     return executeReject("Please set key");
diff --git a/node_modules/branch-cordova-sdk/src/ios/BranchNPM.h b/node_modules/branch-cordova-sdk/src/ios/BranchNPM.h
index 39325ca..8b13789 100644
--- a/node_modules/branch-cordova-sdk/src/ios/BranchNPM.h
+++ b/node_modules/branch-cordova-sdk/src/ios/BranchNPM.h
@@ -1 +1 @@
-#define BRANCH_NPM true
+
diff --git a/node_modules/branch-cordova-sdk/src/ios/BranchSDK.h b/node_modules/branch-cordova-sdk/src/ios/BranchSDK.h
index 3ca1979..2c4547d 100644
--- a/node_modules/branch-cordova-sdk/src/ios/BranchSDK.h
+++ b/node_modules/branch-cordova-sdk/src/ios/BranchSDK.h
@@ -27,6 +27,7 @@
 
 // BranchSDK Basic Methods
 - (void)initSession:(CDVInvokedUrlCommand*)command;
+- (void)initIosSession:(CDVInvokedUrlCommand*)command;
 - (void)disableTracking:(CDVInvokedUrlCommand*)command;
 - (void)setDebug:(CDVInvokedUrlCommand*)command;
 - (void)getAutoInstance:(CDVInvokedUrlCommand*)command;
diff --git a/node_modules/branch-cordova-sdk/src/ios/BranchSDK.m b/node_modules/branch-cordova-sdk/src/ios/BranchSDK.m
index 56e974b..9e529de 100644
--- a/node_modules/branch-cordova-sdk/src/ios/BranchSDK.m
+++ b/node_modules/branch-cordova-sdk/src/ios/BranchSDK.m
@@ -77,6 +77,20 @@ - (void)continueUserActivity:(CDVInvokedUrlCommand*)command
 #pragma mark - Public APIs
 #pragma mark - Branch Basic Methods
 
+- (void)initIosSession:(CDVInvokedUrlCommand*)command
+{
+  Branch *branch = [Branch getInstance];
+  [[Branch getInstance] setDebug];
+
+  id launchOptions = NULL;
+  if (command.arguments.count > 0) {
+    launchOptions = [command.arguments objectAtIndex:0];
+  }
+  [branch delayInitToCheckForSearchAds];
+  [branch useLongerWaitForAppleSearchAds];
+  [branch initSessionWithLaunchOptions:launchOptions andRegisterDeepLinkHandler:^(NSDictionary *params, NSError *error) {}];
+}
+
 - (void)initSession:(CDVInvokedUrlCommand*)command
 {
   [[Branch getInstance] registerPluginName:@"CordovaIonic" version:pluginVersion];
